src/triedb.rs

```diff
@@ -52,12 +55,14 @@ impl<'db> TrieDB<'db> {
     /// Create a new trie with the backing database `db` and `root`
     /// Returns an error if `root` does not exist
     pub fn try_new(db: &'db dyn HashDB, root: &'db H256) -> crate::Result<Self> {
+        let cache: RefCell<LruCache<H256, Vec<u8>>> = RefCell::new(LruCache::new(3000));
```

Comment:
It is an initialization of the cache. In this commit, the cache size is 3000. That means we can store 3,000 key-value pairs.